M2/Macaulay2/tests/normal/methods.m2

```diff
@@ -120,3 +120,10 @@ assert ( chainComplex (new X, FOO => 123 ) === (new OptionTable from {FOO => 123
 assert ( chainComplex (new X) === (new OptionTable from {FOO => BAR},new X from {}) )
 chainComplex X := identity
 assert( chainComplex (new X) === new X )
+
+-- this should list (net/info, HypertextContainer),
+-- even though HypertextContainer is not exported.
+assert(2 == length methods parent class help())
```

Comment:
Was there a reason to call `help()` here and not just use `parent DIV`?  Or if we're trying to avoid importing `DIV`, `code` would work since it also returns a `DIV` object.

Calling `help()` can take a fair amount of memory (I seem to remember an issue about this but I can't find it...) and [one of the PPA builds](https://launchpadlibrarian.net/743391375/buildlog_ubuntu-bionic-i386.macaulay2_1.24.05+git202408112135-0ppa202408100259~ubuntu18.04.1_BUILDING.txt.gz) just crashed here.

Reply:
Oh sure, we can replace this with `code` of something.